Makefile

```diff
@@ -117,6 +120,6 @@ test-e2e-1.19: container-build install-tools
 test-e2e-1.18: container-build install-tools
 	$(call TEST_KUBERNETES,v1.18.0,$(PREFIX),$(VERSION))
 
-test-e2e-all: test-e2e-1.21.1 test-e2e-1.20 test-e2e-1.19 test-e2e-1.18
+test-e2e-all: test-e2e-1.22 test-e2e-1.21.1 test-e2e-1.20 test-e2e-1.19 test-e2e-1.18
 
```

Comment:
Mind removing test-e2e-1.18?

Reply:
For this 1.22 upgrade we are just removing 1.17. I removed it in a different PR that is why it looks like none got removed.

Reply:
👍